airflow/settings.py

```diff
@@ -84,7 +84,7 @@ def timing(cls, stat, dt):
  _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
  """
 
-BASE_LOG_URL = '/admin/airflow/log'
+BASE_LOG_URL = conf.get_url_prefix() + '/admin/airflow/log'
```

Comment:
Same here. Consider using `posixpath.join`

Reply:
I usually use `os.path.join`, is it better to use `posixpath.join` ?

Reply:
While we do not guarantee compatibility with windows per se, os.path.join is os dependent. posixpath.join would be consistent.